CHANGELOG.md

```diff
@@ -10,8 +10,8 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 - Display Gene track beneath each sample track when displaying splice junctions in igv browser
 - Check outdated gene symbols and update with aliases also for cancer
```

Comment:
```suggestion
- Check outdated gene symbols and update with aliases for both RD and cancer variantS
```

Reply:
And feel free to edit that in the v4.4 changelog instead - that would be at least as appropriate. 😁